skimage/filters/thresholding.py

```diff
@@ -621,7 +621,7 @@ def threshold_mean(image):
 
     Parameters
     ----------
-    image : (N, M[, ..., P]) ndarray
+    image : ([P, ..., ]N, M) ndarray
```

Comment:
Well, this `(P, N, M)` is inconsistent with `([P, ] M, N)` above. And anyway I think the ordering of the letters should be alphabetical — the P here does not correspond to "planes" any more than N corresponds to "rows"...